tests/playwright/pages/editor-page.ts

```diff
@@ -95,6 +95,11 @@ export default class EditorPage extends BasePage {
 		}
 	}
 
+	async hideAiPromotion() {
+		// @ts-expect-error Property 'EDITOR_SESSION_ID' exists on window from ai.
+		window.sessionStorage.setItem( 'ai_promotion_introduction_editor_session_key', window.EDITOR_SESSION_ID );
```

Comment:
obviously window is not define in playwright context.
Please use this code from Pro https://github.com/elementor/elementor-pro/blob/dbbb022f0059eef40400e23232b3822714b9ce44/tests/playwright/pages/wp-admin-page.ts#L312
and put it here in core https://github.com/elementor/elementor/blob/fd0c505639dafdf4ad279d4932ab37cd0de8b3d3/tests/playwright/pages/wp-admin-page.ts#L249
We handle pop-ups in this `closeAnnouncementsIfVisible` function